src/main/java/org/zalando/nakadi/partitioning/StringHash.java

```diff
@@ -0,0 +1,18 @@
+package org.zalando.nakadi.partitioning;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class StringHash {
+
+    public int hashCode(final String string) {
```

Comment:
@adyach @rcillo I just extracted String.hashCode to this function, I believe we can't change it without breaking partitioning.